### PR TITLE
Copy edits for gettingStartedContent.ts 

### PR DESCRIPTION
**Pull request by sana-ajani**
-"Open Terminal" is mentioned in the "Primer on GH Codespaces" and the "Learn the fundamentals", if you remove from the Primer, then each section will have 4 tips. 
-Tried to make the capitalization across the buttons and subtext consistent; made the usage of "&" consistent
-Changed "Add more languages & tools support" to "Code in any language, without switching editors". Repl.it's [site](https://repl.it/) similarly mentions "Code in 50+ languages" and I think that notion is more enticing than saying "VS Code supports almost all major languages". The word "support" feels vague.
-Tried to change "Codespaces will give you XYZ" or "VS Code will do XYZ" phrases to be more about actions the user can do, rather than what the editor does or provides.

### CODE REVIEW

File: src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts [hunk 1]
```diff
@@ -85,9 +85,9 @@ export const content: GettingStartedContent = [
 				{
 					id: 'remoteTerminal',
 					title: localize('gettingStarted.remoteTerminal.title', "Run tasks in the integrated terminal"),
-					description: localize('gettingStarted.remoteTerminal.description', "Access your full development environment in the cloud and perform quick command-line tasks."),
+					description: localize('gettingStarted.remoteTerminal.description', "Perform quick command-line tasks using the built-in terminal."),
 					button: {
-						title: localize('gettingStarted.remoteTerminal.button', "Focus Terminal"),
+						title: localize('gettingStarted.remoteTerminal.button', "Open Terminal"),
```
Comment (digitarald) [inline src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts:90]: Terminal is open by default, which is why I ended up with (slightly odd) _"Focus"_. Open works as well though, just might be slightly confusing.

**Comment by digitarald** [comment]
Thank you @sana-ajani 🙏🏻!

> "Open Terminal" is mentioned in the "Primer on GH Codespaces" and the "Learn the fundamentals", if you remove from the Primer, then each section will have 4 tips.

I pondered about this as well! Codespace terminal is special as its the window into the remote machine and doesn't have an easy alternative – vs just opening a terminal app on your local pc. What about hiding the terminal entry in Fundamentals when in Codespaces, so it doesn't duplicate?

> Tried to make the capitalization across the buttons and subtext consistent

Ha, I want back and forth on that as well. Sentence-like capitalization (vs APA headline capitalization) for the headers felt the most conversational, but I don't have strong opinions.

> Changed "Add more languages & tools support" to "Code in any language, without switching editors"

Love that more active language.

**Review by digitarald — APPROVED** [review]
Good to merge. Just had one consideration for "open" vs "focus".

File: src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts (hunk 1 above)
Comment (sana-ajani) [reply src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts:90]: Ah, I see now. I think "Focus" makes sense there then! I saw you merged this PR, so do you want to just update this one?

**Pull request merged**